### DISCUSSION TITLE
How to provide kafka ssl certificate path

### DISCUSSION THREAD

**Discussion by yovelcohen** [milvus-io/milvus#19672] [Q&A and General discussion]
looking at https://github.com/milvus-io/milvus/blob/master/configs/milvus.yaml

It's not specifying how should one pass ssl certificate to milvus' kafka.

**Comment by LoveEachDay** [comment]
@yovelcohen For now we don't provide any config options to pass ssl certificate for kafka.

**Reply by lashsingh** [reply]
@LoveEachDay Given this is over a year old comment, was wondering if it's possible to provide kafka ssl cert path in versions >= 2.3 ? Thanks.
